app/src/main/java/org/groundplatform/android/common/Constants.kt

```diff
@@ -67,7 +67,12 @@ object Constants {
   // Issue URL: https://github.com/google/ground-android/issues/1730
   const val DEFAULT_MOG_TILE_LOCATION = "/offline-imagery/default"
   private const val DEFAULT_MOG_MIN_ZOOM = 8
-  private const val DEFAULT_MOG_MAX_ZOOM = 14
+  const val DEFAULT_MOG_MAX_ZOOM = 14
+  const val DEFAULT_MOG_UPSCALE_MAX_ZOOM = 15
+
+  // These relate to map zoom bounds (float values for runtime rendering)
+  const val DEFAULT_MAP_MIN_ZOOM = 2f
+  const val DEFAULT_MAP_MAX_ZOOM = 21f
```

Comment:
Please refer to the other comment where these are used.

Reply:
it was used earlier, but from last review asked to keep as separate varaible hence created